### ISSUE TITLE
Cad shuts down constantly

### ISSUE THREAD

**Issue by ViolentMinds** [SnailyCAD/snaily-cadv4#1103]
### What version of Node.js are you using?

17.3.0

### What version of npm are you using?

7.10.0

### What operating system are you using?

Windows Server 2019

### What version of SnailyCAD are you currently using?

1.25.3

### Describe the Bug

```
[1] Error occurred during query execution:
[1] ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E40P01), message: "deadlock detected", detail: Some("Process 11840 waits for ShareLock on transaction 579358; blocked by process 9412.\nProcess 9412 waits for ShareLock on transaction 579359; blocked by process 11840."), hint: Some("See server log for query details."), position: None, where_: Some("while updating tuple (9,1) in relation \"Officer\""), schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("deadlock.c"), line: Some(1153), routine: Some("DeadLockReport") }) }) })
[1]     at RequestHandler.handleRequestError (C:\nginx\html\snaily-cadv4\node_modules\@prisma\client\runtime\index.js:29913:13)
[1]     at RequestHandler.request (C:\nginx\html\snaily-cadv4\node_modules\@prisma\client\runtime\index.js:29892:12)
[1]     at async PrismaClient._request (C:\nginx\html\snaily-cadv4\node_modules\@prisma\client\runtime\index.js:30864:16)
[1]     at async Promise.all (index 0)
[1]     at async setInactiveUnitsOffDuty (C:\nginx\html\snaily-cadv4\packages\api\src\lib\leo\setInactiveUnitsOffDuty.ts:6:3)
[1] yarn workspace @snailycad/api generate && yarn workspace @snailycad/api start exited with code 1
```

### Expected Behavior

This randomly happens it seems? Not sure what to do

**Comment by Dev-CasperTheGhost** [comment]
Duplicate of #1093. This should've been fixed already, but I'll take another look later today. This error occurred after 1.25.3?
